src/store/actions/api.actions.js

```diff
@@ -376,33 +376,32 @@ export const preparePlugins = api => api.plugins.map(plugin => {
     }
 });
 
-export const saveEndpoint = (pathname, api) => dispatch => {
-    dispatch(openConfirmationModal(
-        'save',
-        () => confirmedSaveEndpoint(dispatch, pathname, api),
-        api.name,
-    ));
+export const saveEndpoint = api => dispatch => {
```

Comment:
I'd consider no brackets here, but feel free to decide what is more suitable
```js
export const saveEndpoint = api => dispatch =>
   dispatch(openConfirmationModal('save', api, api.name));
```

Reply:
`genau` !